index.js

```diff
@@ -16,10 +16,22 @@ var Guard = function (options) {
   this._options = xtend(defaults, options)
 }
 
+function isString (value) {
+  return typeof value === 'string'
+}
+
+function isArray (value) {
+  return value instanceof Array
```

Comment:
Why not Array.isArray?

Reply:
Looks like the only difference is that `Array.isArray` works across iframes.
(https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray#instanceof_vs_isArray)

Not sure if there are any microbenchmarks about performance though.